[2025-06-27T02:06:12.266+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T02:06:12.287+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [queued]>
[2025-06-27T02:06:12.295+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [queued]>
[2025-06-27T02:06:12.295+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T02:06:12.315+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-23 02:07:00+00:00
[2025-06-27T02:06:12.353+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=82) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T02:06:12.357+0000] {standard_task_runner.py:72} INFO - Started process 94 to run task
[2025-06-27T02:06:12.339+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt', 'create_ga4_events_dataset', 'scheduled__2025-06-23T02:07:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpt_i5prng']
[2025-06-27T02:06:12.366+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask create_ga4_events_dataset
[2025-06-27T02:06:12.470+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [running]> on host 040d1cf847a3
[2025-06-27T02:06:12.626+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-23T02:07:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-23T02:07:00+00:00'
[2025-06-27T02:06:12.633+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T02:06:12.657+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T02:06:12.678+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T02:06:12.684+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events.
[2025-06-27T02:06:12.685+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T02:06:12.686+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events in project: sixth-topic-349709 
[2025-06-27T02:06:14.502+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T02:06:14.547+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T02:06:14.548+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-23T02:07:00+00:00, execution_date=20250623T020700, start_date=20250627T020612, end_date=20250627T020614
[2025-06-27T02:06:14.606+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T02:06:14.667+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T02:06:14.672+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T02:17:48.660+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T02:17:48.679+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [queued]>
[2025-06-27T02:17:48.689+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [queued]>
[2025-06-27T02:17:48.692+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T02:17:48.720+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-23 02:07:00+00:00
[2025-06-27T02:17:48.733+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=88) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T02:17:48.739+0000] {standard_task_runner.py:72} INFO - Started process 105 to run task
[2025-06-27T02:17:48.746+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt', 'create_ga4_events_dataset', 'scheduled__2025-06-23T02:07:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpydxwhsha']
[2025-06-27T02:17:48.755+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask create_ga4_events_dataset
[2025-06-27T02:17:48.876+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [running]> on host ece8b7d95be1
[2025-06-27T02:17:48.965+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-23T02:07:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-23T02:07:00+00:00'
[2025-06-27T02:17:48.967+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T02:17:48.981+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T02:17:48.997+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T02:17:49.003+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events.
[2025-06-27T02:17:49.004+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T02:17:49.005+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events in project: sixth-topic-349709 
[2025-06-27T02:17:50.826+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T02:17:50.851+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T02:17:50.853+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-23T02:07:00+00:00, execution_date=20250623T020700, start_date=20250627T021748, end_date=20250627T021750
[2025-06-27T02:17:50.888+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T02:17:50.908+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T02:17:50.908+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T02:21:09.201+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T02:21:09.239+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [queued]>
[2025-06-27T02:21:09.256+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [queued]>
[2025-06-27T02:21:09.265+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T02:21:09.292+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-23 02:07:00+00:00
[2025-06-27T02:21:09.309+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=85) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T02:21:09.317+0000] {standard_task_runner.py:72} INFO - Started process 110 to run task
[2025-06-27T02:21:09.326+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt', 'create_ga4_events_dataset', 'scheduled__2025-06-23T02:07:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpclrgt9k1']
[2025-06-27T02:21:09.336+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask create_ga4_events_dataset
[2025-06-27T02:21:09.431+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [running]> on host a6d750376b6f
[2025-06-27T02:21:09.527+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-23T02:07:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-23T02:07:00+00:00'
[2025-06-27T02:21:09.536+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T02:21:09.548+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T02:21:09.560+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T02:21:09.562+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events.
[2025-06-27T02:21:09.566+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T02:21:09.567+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events in project: sixth-topic-349709 
[2025-06-27T02:21:11.325+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T02:21:11.384+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T02:21:11.386+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-23T02:07:00+00:00, execution_date=20250623T020700, start_date=20250627T022109, end_date=20250627T022111
[2025-06-27T02:21:11.455+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T02:21:11.502+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-27T02:21:11.505+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T02:34:53.255+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T02:34:53.331+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [queued]>
[2025-06-27T02:34:53.340+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [queued]>
[2025-06-27T02:34:53.344+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T02:34:53.379+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-23 02:07:00+00:00
[2025-06-27T02:34:53.403+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=86) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T02:34:53.408+0000] {standard_task_runner.py:72} INFO - Started process 110 to run task
[2025-06-27T02:34:53.408+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt', 'create_ga4_events_dataset', 'scheduled__2025-06-23T02:07:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmp21ij75sc']
[2025-06-27T02:34:53.422+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask create_ga4_events_dataset
[2025-06-27T02:34:53.544+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-23T02:07:00+00:00 [running]> on host ba60ab6371e3
[2025-06-27T02:34:53.715+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-23T02:07:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-23T02:07:00+00:00'
[2025-06-27T02:34:53.722+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T02:34:53.738+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T02:34:53.741+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T02:34:53.744+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events.
[2025-06-27T02:34:53.746+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T02:34:53.750+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events in project: sixth-topic-349709 
[2025-06-27T02:34:55.378+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T02:34:55.440+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T02:34:55.442+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-23T02:07:00+00:00, execution_date=20250623T020700, start_date=20250627T023453, end_date=20250627T023455
[2025-06-27T02:34:55.506+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T02:34:55.575+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-27T02:34:55.588+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
